module/VuFindTheme/src/VuFindTheme/View/Helper/HeadLink.php

```diff
@@ -114,7 +114,7 @@ protected function getFileType()
     public function itemToString(stdClass $item)
     {
         // Normalize href to account for themes (if appropriate), then call the parent class:
-        if ($this->isRelativePath($item->href)) {
+        if ($this->isRelativePath($item->href ?? '')) {
```

Comment:
I think it would actually be better to make this `if (isset($item->href) && $this->isRelativePath($item->href)) {`... Otherwise, passing the blank string to `isRelativePath` will lead to a return value of true, which will lead to further unnecessary attempts to access an undefined value.

Reply:
yeah, will do

Reply:
Hmm, ill try to change the base also :) havent really done it before, but learning is the key to success

Reply:
Decided to just open new pr created from release-10 branch